slides/syntroduction/Dockerfile

```diff
@@ -20,11 +20,12 @@ COPY --from=htmlmaker /presentation/node_modules/lato-font /slides/node_modules/
 COPY --from=htmlmaker /presentation/node_modules/typeface-ubuntu /slides/node_modules/typeface-ubuntu
 COPY --from=htmlmaker /presentation/node_modules/typeface-ubuntu-mono /slides/node_modules/typeface-ubuntu-mono
 COPY --from=htmlmaker /presentation/node_modules/@fortawesome /slides/node_modules/@fortawesome
-RUN node /decktape/decktape.js --chrome-path chromium-browser --chrome-arg=--no-sandbox --size '2560x1440' --pause 2000 --chrome-arg=--allow-file-access-from-files /slides/slides.html /slides/slides.pdf
+COPY --from=htmlmaker /presentation/node_modules/@fontsource /slides/node_modules/@fontsource
+RUN node /decktape/decktape.js --chrome-path chromium-browser --chrome-arg=--no-sandbox --size '1920x1080' --pause 2000 --chrome-arg=--allow-file-access-from-files /slides/slides.html /slides/slides.pdf
```

Comment:
Was the resolution change done by accident?

Reply:
No, that's according to the latest (internal) cookiecutter template for creating Asciidoctor slides